content/topics/javascript-specific/js-naming-convention/_index.md

```diff
@@ -0,0 +1,56 @@
+---
+content_type: topic
+prerequisites:
+  hard:
+    - topics/file-and-directory-naming/general 
```

Comment:
Kindle merge develop into your branch and update the general file accordingly

Reply:
It's merged @Rokhuda